### PR TITLE
Example documentation

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #12

### CODE REVIEW

File: docs/api-assetmanager.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
+
+<p align="center">
+    <h1 align="center">Api Library AssetManager</h1>
+</p>
+
+The **AssetManager** is a library that allows you to **generate the configuration of all your css and javascript assets easily**, and then generate your html code.
+
+### setAppendTimestamp:
```
Comment (samdark) [inline docs/api-assetmanager.md]: API docs should stay in phpdoc. There's no need to have these in readme.

File: docs/assetbundles.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
+
+<p align="center">
+    <h1 align="center">Defined Asset Bundles</h1>
+</p>
+
+AssetBundle represents a collection of asset files, such as css, js, images.
```
Comment (samdark) [inline docs/assetbundles.md]:
```suggestion
AssetBundle represents a collection of asset files, such as CSS, JavaScript, images.
```

File: docs/assetbundles.md [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+|`$css`|`array`| List of css files that this bundle contains.| `[]`|
+|`$cssOptions`|`array`| The options that will be passed to \Yiisoft\View\WebView::setCssFiles().| `[]`|
+|`$depends`|`array`| List of bundle class names that this bundle depends on.| `[]`|
+|`$js`|`array`| List of javascript files that this bundle contains.| `[]`|
+|`$jsOptions`|`array`| The options that will be passed to \Yiisoft\View\WebView::setJsFiles().| `[]`|
+|`$publishOptions`|`array`| The options to be passed to \Yiisoft\Assets\AssetPublisher::publish() when the asset bundle is being published.|`[]`|
+|`$sourcePath`|`string/null`| The directory that contains the source asset files for this assetBundle.| `null`|
+
+
+## 1.- Example AssetBundle with sourcePath:
```
Comment (samdark) [inline docs/assetbundles.md]: This part is awesome 👍

File: README.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
     <a href="https://github.com/yiisoft" target="_blank">
         <img src="https://github.com/yiisoft.png" height="100px">
     </a>
-    <h1 align="center">Yii assets</h1>
+    <h1 align="center">Yii Library Assets</h1>
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply README.md]: Remove.

File: README.md [hunk 5]
```diff
@@ -14,5 +14,53 @@ The package impements clientside asset management such as CSS and JavaScript.
...
+composer require yiisoft/assets
+~~~
+
+Or add to composer.json:
+
+~~~
+"yiisoft/assets": "^1.0@dev"
+~~~
+
+### HOW TO CONFIG ASSETMANAGER: ###
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply README.md]: Done.

File: README.md [hunk 6]
```diff
@@ -14,5 +14,53 @@ The package impements clientside asset management such as CSS and JavaScript.
...
 
+### REQUIREMENTS:
+
+- The minimum requirement by this project template that your Web server supports:
+    - PHP 7.4 or higher.
+    - [Composer Config Plugin](https://github.com/hiqdev/composer-config-plugin)
+
+### HOW TO INSTALL:
+
+<p>
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply README.md]: Done.

File: docs/assetbundles.md (hunk 2 above)
Comment (terabytesoftw) [reply docs/assetbundles.md]: Done.

File: docs/api-assetmanager.md (hunk 1 above)
Comment (terabytesoftw) [reply docs/api-assetmanager.md]: Move docs, Done.

File: docs/assetbundles.md (hunk 3 above)
Comment (terabytesoftw) [reply docs/assetbundles.md]: Thks.

File: docs/assetbundles.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+|`$basePath`|`string/null`| The web public directory that contains the asset files in this bundle| `null`|
+|`$baseUrl`|`string/null`| The base URL for the relative asset files listed in $js and $css.| `null`|
+|`$cdn`|`bool`| Indicate if we are going to use cdn exclusively.| `false`|
+|`$css`|`array`| List of css files that this bundle contains.| `[]`|
+|`$cssOptions`|`array`| The options that will be passed to \Yiisoft\View\WebView::setCssFiles().| `[]`|
+|`$depends`|`array`| List of bundle class names that this bundle depends on.| `[]`|
+|`$js`|`array`| List of javascript files that this bundle contains.| `[]`|
+|`$jsOptions`|`array`| The options that will be passed to \Yiisoft\View\WebView::setJsFiles().| `[]`|
+|`$publishOptions`|`array`| The options to be passed to \Yiisoft\Assets\AssetPublisher::publish() when the asset bundle is being published.|`[]`|
+|`$sourcePath`|`string/null`| The directory that contains the source asset files for this assetBundle.| `null`|
```
Comment (samdark) [inline docs/assetbundles.md]:
```suggestion
|`$sourcePath`|`string/null`| The directory that contains the source asset files for this asset bundle.| `null`|
```

File: docs/assetmanager.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
+# AssetManager
+
+AssetManager manages asset bundle configuration and loading.
+
+## 1.- Public Methods:
```
Comment (samdark) [inline docs/assetmanager.md]: Should be moved to phpdoc. No need to define it here.

File: README.md [hunk 9]
```diff
@@ -14,5 +14,43 @@ The package impements clientside asset management such as CSS and JavaScript.
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/yiisoft/assets/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/assets/?branch=master)
 [![Code Coverage](https://scrutinizer-ci.com/g/yiisoft/assets/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/assets/?branch=master)
 
-## General usage
 
+### Requirements:
+
+- The minimum requirement by this project template that your Web server supports:
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply README.md]: Done.

File: README.md [hunk 10]
```diff
@@ -14,5 +14,53 @@ The package impements clientside asset management such as CSS and JavaScript.
...
+If you do not have <a href="http://getcomposer.org/" title="Composer" target="_blank">Composer</a>, you may install it by following the instructions at <a href="http://getcomposer.org/doc/00-intro.md#installation-nix" title="getcomposer.org" target="_blank">getcomposer.org</a>.
+</p>
+
+composer:
+
+~~~
+composer require yiisoft/assets
+~~~
+
+Or add to composer.json:
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply README.md]: Remove. done.

File: docs/api/assetmanager/registerjsfile.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
+### AssetManager::registerJsFile()
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply docs/api/assetmanager/registerjsfile.md]: Remove. done.

File: docs/assetmanager.md (hunk 8 above)
Comment (terabytesoftw) [reply docs/assetmanager.md]: Remove. done

**Pull request merged**